src/ucc/api/ucc.h

```diff
@@ -903,7 +903,7 @@ enum ucc_team_params_field {
     UCC_TEAM_PARAM_FIELD_OUTSTANDING_COLLS      = UCC_BIT(1),
     UCC_TEAM_PARAM_FIELD_EP                     = UCC_BIT(2),
     UCC_TEAM_PARAM_FIELD_EP_LIST                = UCC_BIT(3),
-    UCC_TEAM_PARAM_FIELD_EP_TYPE                = UCC_BIT(4),
+    UCC_TEAM_PARAM_FIELD_EP_RANGE               = UCC_BIT(4),
```

Comment:
? I’m guessing it is because of naming consistency. If so, you might want to make params and attribs consistent.

Reply:
correct, will fix attribs as well